dist/diffhtml-runtime.js

```diff
@@ -827,49 +835,63 @@ function patchNode(node, patches) {
 
           // Attribute manipulation.
           else if (patch.__do__ === _sync.MODIFY_ATTRIBUTE) {
-              var attrChangePromises = (0, _transitions.makePromises)('attributeChanged', [el], patch.name, el.getAttribute(patch.name), patch.value);
+              var attributes = patch.attributes;
 
-              triggerTransition('attributeChanged', attrChangePromises, function (promises) {
-                var callback = function callback() {
-                  // Remove.
-                  if (patch.value === undefined) {
-                    el.removeAttribute(patch.name);
+              attributes.forEach(function (attr) {
+                var attrChangePromises = (0, _transitions.makePromises)('attributeChanged', [el], attr.name, el.getAttribute(attr.name), attr.value);
 
-                    if (patch.name in el) {
-                      el[patch.name] = undefined;
+                triggerTransition('attributeChanged', attrChangePromises, function (promises) {
+                  var callback = function callback() {
+                    // Release the Virtual Tree Attribute allocations.
+                    if (patch.toRemove) {
+                      patch.toRemove.forEach(_pools.pools.attributeObject.unprotect);
                     }
-                  }
-                  // Change attributes.
-                  else {
-                      var isObject = _typeof(patch.value) === 'object';
-                      var isFunction = typeof patch.value === 'function';
-
-                      // If not a dynamic type, set as an attribute, since it's a valid
-                      // attribute value.
-                      if (!isObject && !isFunction) {
-                        if (patch.name) {
-                          el.setAttribute(patch.name, patch.value);
-                        }
-                      } else if (typeof patch.value !== 'string') {
-                        // Necessary to track the attribute/prop existence.
-                        el.setAttribute(patch.name, '');
 
-                        // Since this is a dynamic value it gets set as a property.
-                        el[patch.name] = patch.value;
-                      }
+                    // Remove the attribute or property.
+                    if (!attr.value) {
+                      el.removeAttribute(attr.name);
```

Comment:
Real element. I removed the `getAttribute` call before, so this should not be an issue now.